articles/concepts/Glossary.md

```diff
@@ -12,31 +12,33 @@ uid: microsoft.quantum.glossary
+|<a id="operation"></a>Operation|The basic unit of quantum execution in Q#. It is roughly equivalent to a function in C, C++ or Python, or a static method in C# or Java. For more information, see [Operation and function types](xref:microsoft.quantum.language.type-model#operation-and-function-types).|
+|<a id="operator-application"></a>Operator application|Performing a quantum operation. This typically applies a unitary matrix to the current quantum   state vector.|
+|<a id="oracle"></a>Oracle|A subroutine that provides data-dependent information to a quantum algorithm at runtime. Typically, the goal is to provide a superposition of outputs corresponding to inputs that are in superposition. For more information, see [Oracles](xref:microsoft.quantum.libraries.data-structures#oracles).|
+|<a id="partial-app"></a>Partial application|Calling a function or operation without all the required parameters. The returns a new callable that only needs the missing parameters supplied during a future application. For more information, see [Partial application](xref:microsoft.quantum.libraries.data-structures#partial-application).|
+|<a id="pauli"></a>Pauli operators|The `X`, `Y` and `Z` quantum gates. For more information, see [Single-qubit operations](xref:microsoft.quantum.concepts.qubit#single-qubit-operations).|
+|<a id="prelude"></a>Prelude|The set of primitive and classical operations and functions defined by each individual target machine, rather than at the Q# level For more information, see [The prelude](xref:microsoft.quantum.libraries.standard.prelude).|
+|<a id="circuit"></a>Quantum circuit|A diagrammatical representation of a program for a quantum computer. FOr more information, see [Quantum circuits](xref:microsoft.quantum.concepts.circuits).|
+|<a id="quantum-state"></a>Quantum state|A representation of the qubits in a system, usually denoted as a complex column vector called the *quantum state vector*. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|
+|<a id="qubit"></a>Qubit|A basic unit of quantum storage. For more information, see [The Qubit](xref:microsoft.quantum.concepts.qubit).|
+|<a id="rus"></a>Repeat-until-success|A quantum algorithm that probabilistically succeeds. Upon failure, the routine will re-try until successful (or a limit has been reached). For more information, see [Repeat ntil Success (RUS)](xref:microsoft.quantum.techniques.qubits#measurements)|
+|<a id="standard"></a>Standard|Operations and functions defined in Q# that build on the logic defined in the [prelude](xref:microsoft.quantum.glossary#prelude). The standard library implementation is agnostic with respect to target machines. For more information, see [Standard libraries](xref:microsoft.quantum.libraries.standard.intro).|
```

Comment:
This may need modified slightly if you decide to adopt my above suggestion about removing the "prelude" entry.